vars/main.yml

```diff
@@ -15,6 +15,19 @@ nms_state_vals:
 # Package installation state
 nms_state: "{{ nms_state_vals[nms_setup] | default(nms_state_vals[nms_default_setup]) }}"
 
+# Service started/stopped state variables
+nms_service_state_vars: [
+  stopped, started, restarted, reloaded
+]
+
+nms_default_service_state: started
```

Comment:
Maybe we can get rid of `nms_default_service_state` and `nms_service_state_vals` since it doesn't look like we need to translate the input to another values that isn't a normal ansible state value?

Reply:
Good call, this has been updated.